src/ProductConstructionService/ProductConstructionService.DependencyFlow/PullRequestUpdater.cs

```diff
@@ -130,18 +130,18 @@ public async Task<bool> UpdateAssetsAsync(
                 Assets = assets,
                 IsCoherencyUpdate = false,
             },
-            checkNextCommitToProcess);
+            forceApply);
     }
 
     /// <summary>
     ///     Process any pending pull request updates.
     /// </summary>
-    /// <param name="checkNextCommitToProcess">If true, we will check if this commit is the latest one we have queued. If it's not we will skip this update.
-    ///                                         Should be set to false when calling from SubscriptionTriggerer </param>
+    /// <param name="forceApply">If false, we will check if this build is the latest one we have queued. If it's not we will skip this update.
+    ///                                         Should be set to true when calling from SubscriptionTriggerer </param>
```

Comment:
I'd rather not claim this on the API side. We can put a comment on the caller side explaining it if we need to. But it seems off to have it on the callee side.

```suggestion
    /// <param name="forceApply">If false, we will check if this build is the latest one we have queued. If it's not we will skip this update.</param>
```